src/helpers/scriptRunner.js

```diff
@@ -1,15 +1,18 @@
 import FormContext from './FormContext';
+import { httpInterceptor } from '../helpers/httpInterceptor';
 
 export default class ScriptRunner {
 
   constructor(rootRecord, patient, parentRecord) {
     this.formContext = new FormContext(rootRecord, patient, parentRecord);
+    this.interceptor = httpInterceptor;
   }
 
   execute(eventJs) {
     const formContext = this.formContext;
-    if (eventJs) {
-      const executiveJs = `(${eventJs})(formContext)`;
+    const interceptor = this.interceptor;
```

Comment:
Please add details of how to use interceptor in product card, as example

Reply:
okay Will update the card